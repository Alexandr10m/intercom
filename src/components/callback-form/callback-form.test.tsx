import * as React from "react";
import * as renderer from "react-test-renderer";
import CallBackForm from "./callback-form";


describe(`CallBackForm Snapshots`, () => {
  it(`should render correct`, () => {
    const tree = renderer
      .create(
        <CallBackForm
          onSubmit = {() => {}}
          onInput = {() => {}}
          buttonDisable = {true}
          inputValue = {`2`}
        />
        )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

