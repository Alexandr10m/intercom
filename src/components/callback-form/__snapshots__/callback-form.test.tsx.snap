// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CallBackForm Snapshots should render correct 1`] = `
<form
  action="#"
  autoComplete="on"
  className="header_callback-form"
  encType="multipart/form-data"
  id="callback-Form"
  method="POST"
  name="callback"
>
  <p
    className="header__excerpt"
  >
    Узнайте стоимость вашего бетонного бассейна
  </p>
  <div
    className="callback__text-wrap"
  >
    <input
      className="header__number-phone"
      id="number-phone"
      name="number-phone"
      onChange={[Function]}
      placeholder="+38 (093) 333 33 33"
      type="text"
      value="2"
    />
    <button
      className="header__button"
      disabled={true}
      onClick={[Function]}
      type="submit"
    >
      Узнать стоимость
    </button>
  </div>
</form>
`;
